boards/modalai/fc-v1/nuttx-config/nsh/defconfig

```diff
@@ -257,9 +255,7 @@ CONFIG_USART2_BAUD=57600
 CONFIG_USART2_IFLOWCONTROL=y
 CONFIG_USART2_OFLOWCONTROL=y
 CONFIG_USART2_RXBUFSIZE=600
-CONFIG_USART2_RXDMA=y
 CONFIG_USART2_TXBUFSIZE=3000
-CONFIG_USART2_TXDMA=y
```

Comment:
I believe it's only this one (USART2_TXDMA) that's the problem. https://github.com/PX4/PX4-Autopilot/pull/20956#issuecomment-1400435251